### PR TITLE
Fix TA bug

### PR DESCRIPTION
**Pull request by bmcdanie**
actually fix it this time - thanks @tyounkin

### CODE REVIEW

**Comment by bmcdanie** [comment]
test this please

File: src/time_advance_tests.cpp [hunk 1]
```diff
@@ -28,7 +28,10 @@ template<typename P>
 void time_advance_test(int const level, int const degree, PDE<P> &pde,
                        int const num_steps, std::string const filepath,
                        bool const full_grid                            = false,
-                       std::vector<std::string> const &additional_args = {})
+                       std::vector<std::string> const &additional_args = {},
+                       double const tol_factor                         = 1e4)
```
Comment (grahamlopez) [inline src/time_advance_tests.cpp]: this gets set locally as `eps_multiplier` down below. should this default parameter become `eps_multiplier`?

Also, other places in the file that call this are using `tol_fac` locally. probably those need to change as well.

Comment (bmcdanie) [reply src/time_advance_tests.cpp]: this is something weird git did with the rebase, should be fixed.

**Comment by bmcdanie** [comment]
test this please

**Comment by grahamlopez** [comment]
test this please

**Pull request merged**